style.css

```diff
@@ -0,0 +1,416 @@
+	padding: 20px 0;
+	margin-bottom: 50px;
+}
+
+.navigation {
+	font-size: 0;
+	width: 80%;
+	margin: 0 auto;
+	display: block;
+	text-align: center;
+}
+
+.navigation > li {
+	margin-left: 5%;
+	font-size: 11px;
+	display: inline-block;
+	line-height: 28px;
+}
+
+.navigation > li:first-child {
+	margin-left: 0;
+}
+
+.navigation > li a {
+	text-decoration: none;
+	text-transform: uppercase;
+}
+
+.has-submenu {
+	position: relative;
+}
+
+.sub-navigation {
+	display: none;
+	position: absolute;
+	top: 100%;
+	left: -15px;
+	background: #ffffff;
+	padding: 15px;
+	line-height: 12px;
+	box-shadow: 2px 2px 2px rgba(0, 0, 0, 0.25);
+}
+
+.sub-navigation li {
+	text-align: left;
+}
+
+.sub-navigation li+li {
+	margin-top: 10px;
+}
+
+.navigation > li:hover .sub-navigation {
+	display: block;
+}
+
+.has-submenu > a {
+	display: inline-block;
+	background: url(images/arrow-down.png) right 11px no-repeat;
+	padding-right: 16px;
+}
+
+.utility-menu {
+	float: right;
+	font-size: 0;
+}
+
+.utility-menu li {
+	display: inline-block;
+}
+
+.utility-menu li+li {
+	border-left: 1px dotted #999999;
+}
+
+.utility-menu-wishlist,
+.utility-menu-search {
+	background-image: url(images/sprite-utility-menu.png);
+	background-repeat: no-repeat;
+	display: inline-block;
+	padding: 14px 22px;
+}
+
+.utility-menu-wishlist {
+	background-position: 14px center;
+}
+
+.utility-menu-search {
+	background-position: -34px center;
+}
+
+.call-advertising {
+	background: #d9e1e6;
+	width: 230px;
+	position: fixed;
+	bottom: 0;
+	left: 0;
+	padding: 2px 2px;
+	text-align: center;
+}
+
+.call-advertising-number {
+	display: none;
+	background: #d9e1e6;
+}
+
+.call-advertising:hover .call-advertising-number {
+	display: block;
+}
+
+.call-advertising-number-why {
+	display: inline-block;
+}
+
+.call-advertising-because {
+	display: none;
+	font-size: 12px;
+	width: 130px;
+	background: #FFFF00;
+	position: absolute;
+	left: 230px;
+	bottom: 15px;
+}
+
+.call-advertising-number-why:hover .call-advertising-because {
+	display: block;
+}
+
+.product-image-container {
+	width: 60%;
+	float: left;
+}
+
+.product-details-container {
+	width: 40%;
+	float: right;
+}
+
+.product-redirect {
+	padding-top: 10px;
+	font-size: 14px;
+	margin-bottom: 40px;
+}
+
+.product-category-link {
+	color: #68899a;
+	text-decoration: none;
+}
+
+.product-price {
+	font-size: 24px;
+	color: #68899a;
+	margin-bottom: 25px;
+}
+
+.product-name {
+	font-size: 24px;
+	margin-top: 0;
+	margin-bottom: 18px;
+}
+
+.product-rating-review {   /* todo: need style for this block, i get back to this point later */
+	padding-bottom: 20px;
+}
+
+.product-additional-info-availability {
+	font-size: 14px;
+}
+
+.product-additional-info-property-value {
+	color: #68899a;
+}
+
+.product-tag {
+	color: #68899a;   /* zzhanna4796 */
+	text-decoration: none;
+}
+
+.product-additional-info-availability {
+	margin-top: 10px;
+	font-weight: bold;
+}
+
+.product-description {
+	margin: 27px 0 40px;
```

Comment:
why do you use margin-top - remember - whenever possible - use margin-bottom to place blocks under each other